libvirt-glib.rb

```diff
@@ -0,0 +1,105 @@
+class LibvirtGlib < Formula
+  desc "GLib and GObject mapping for libvirt"
+  homepage "http://www.libvirt.org"
+  url "http://libvirt.org/sources/glib/libvirt-glib-0.2.3.tar.gz"
+  sha256 "c3b11aa34584b3416148b36bb005505e461830c73c9fd2f40143cca3309250dd"
+
+  depends_on "gettext" => :build
+  depends_on "intltool" => :build
+  depends_on "pkg-config" => :build
+
+  depends_on "libvirt"
+
+  depends_on "gobject-introspection" => :optional
+  depends_on "vala" => :optional
+
+  # work around ld not understanding --version-script argument
+  # upstream bug: https://bugzilla.redhat.com/show_bug.cgi?id=1304981
+  patch :DATA
+
+  def install
+    args = ["--prefix=#{prefix}",
+            "--localstatedir=#{var}",
+            "--mandir=#{man}",
+            "--sysconfdir=#{etc}",
+            "--enable-installed-tests",
+            "--enable-always-build-tests",
+            "--with-python"]
+
+    args << "--enable-vala" if build.with? "vala"
+    args << "--enable-introspection" if build.with? "gobject-introspection"
+
+    system "./configure", *args
+
+    # Compilation of docs doesn"t get done if we jump straight to "make install"
```

Comment:
Addressed in https://github.com/vicamo/homebrew/commit/ab33b991908a987a8c9d12bbe600dc7744ea913a